dts/arm/silabs/xg28/xg28.dtsi

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2025 Christoph Jans
+ * Copyright (c) 2025 Shontal Biton
```

Comment:
Since all your changes to this file should be removed, please remember to remove this too :)